src/test/java/school/hei/haapi/integration/FeeIT.java

```diff
@@ -83,29 +95,61 @@ void teacher_read_ko() {
   @Test
   void manager_read_ok() throws ApiException {
     ApiClient manager1Client = anApiClient(MANAGER1_TOKEN);
-
     PayingApi api = new PayingApi(manager1Client);
-    Fee actual = api.getStudentFeeById(STUDENT1_ID, FEE1_ID);
 
-    assertEquals(fee1(), actual);
+    List<Fee> actual = api.getStudentFees(STUDENT1_ID);
+
+    assertTrue(actual.contains(fee1()));
   }
 
   @Test
-  void student_read_ko() {
+  void student1_read_student2_fees_ko() {
```

Comment:
revert. `student_read_ko` means cannot read in general, and is consistent with naming of other tests